apps/api/src/common/filters/http-exception.filter.ts

```diff
@@ -40,5 +40,11 @@ export class HttpExceptionFilter implements ExceptionFilter {
         path: request.url,
       });
     }
+    this.logger.error(
+      JSON.stringify({
+        request: { method: request.method, url: request.url },
+        exception,
+      }),
+    );
```

Comment:
I think we decided to log exceptions by logger's default setting (ref: https://github.com/line/abc-user-feedback/blob/3b7a8add1a14116d1b1ee2d020ba03feb04a546d/apps/api/src/app.module.ts#L95-L105)